test/powershell/Modules/Microsoft.PowerShell.Security/CmsMessage.Tests.ps1

```diff
@@ -124,9 +151,12 @@ Describe "CmsMessage cmdlets thorough tests" -Tags "Feature" {
     BeforeAll {
         if ($IsWindows)
         {
-            $certLocation = Create-TestCertificate
+            $certLocation = Create-GoodCertificate
             $certLocation | Should Not BeNullOrEmpty | Out-Null
 
+            $badCertLocation = Create-BadCertificate
+            $badCertLocation | Should Not BeNullOrEmpty | Out-Null
+
             if ($IsCoreCLR)
             {
                 # PKI module is not available for PowerShell Core, so we need to use Windows PowerShell to import the cert
```

Comment:
I have verified that it works fine on NanoServer, so thus I assume the same on IoT. Here is a screenshot:

![image](https://cloud.githubusercontent.com/assets/127450/23675916/c03f3686-032f-11e7-8344-dcfbefbf0174.png)